llmfoundry/callbacks/model_gauntlet_callback.py

```diff
@@ -45,7 +46,7 @@ class ModelGauntlet(Callback):
     def __init__(self,
                  logger_keys: dict,
                  categories: dict,
-                 weighting: Weighting = Weighting.EQUAL,
+                 weighting: str = 'EQUAL',
```

Comment:
This is a breaking API change -- can we avoid this?

Reply:
So, this callback has a couple issues that mean I strongly suspect no one is using this as is (see the comment in `compute_averages` for the other major issue). One of the issue is that this default actually doesn't work. It would then try to access weighting via `Weighting[Weighting.EQUAL]` which is not a valid way to access an enum. Given that we mostly instantiate via yaml, I assume this was getting instantiated via a string from a yaml and the default has never been run.

Reply:
OK -- can you file a JIRA ticket for @bmosaicml to fix?

Reply:
Yeah, I think he already knows/is discussing with Evan how to implement this in a better way. @bmosaicml is there already a jira?

Reply:
https://mosaicml.atlassian.net/browse/RESEARCH-801